api/machines.go

```diff
@@ -56,3 +57,32 @@ func createTokenSecret() (*base64.Value, error) {
 
 	return base64.NewValue(value), nil
 }
+
+// List machines in the given org and state
+func (m *MachinesClient) List(ctx context.Context, orgIDs []*identity.ID, state *string, teamID *identity.ID) ([]*apitypes.MachineSegment, error) {
```

Comment:
The endpoint only supports a single org id today, so we should probably match that behaviour